notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    #lets print out the first 10 entries
</pre><p>It looks like the template and example (https://sciencebetter.slack.com/archives/C03FDR44L07/p1660233388139449) typically use Markdown cells to explain what code blocks will be doing, rather than comments. (Admittedly sometimes comments are used in code blocks.) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>